requirements.txt

```diff
@@ -8,6 +8,10 @@ idna==2.9
 itsdangerous==1.1.0
 Jinja2==2.11.1
 MarkupSafe==1.1.1
+pandas==1.0.3
```

Comment:
Since we aren’t actually using it anymore, it might make sense to drop this dependency for now.